src/c_api.cpp

```diff
@@ -728,6 +728,7 @@ class Booster {
 
   std::string SaveModelToString(int start_iteration, int num_iteration,
                                 int feature_importance_type) const {
+
```

Comment:
Please revert this and any other whitespace-only changes not related to this PR. Such changes increase the size of the diff and therefore the effort required to review.